server/server.js

```diff
@@ -1,11 +1,40 @@
 const express = require('express');
-
 const app = express();
+
 const path = require('path');
+
+const cookies = require('cookie-session');
+const bodyParser = require('body-parser');
+
+//routers
+const login = require('./router/login');
+const signin = require('./router/signin');
+const aquarium = require('./router/aquarium');
```

Comment:
Maybe it would be nicer to be more specific in naming such as loginRouter, signinRouter, aquariumRouter?